backend/inspirehep/records/marshmallow/authors/ui.py

```diff
@@ -39,19 +39,19 @@ def get_facet_author_name(self, data):
 
     @staticmethod
     def get_twitter(data):
-        return get_id_for_schema(data, "TWITTER")
+        return get_id_for_schema(data, "ids", "TWITTER")
```

Comment:
I'd go with `kwargs` here, `ids_field=..., schema=...`  and actually default `ids_field` to `ids`,

or move out `data.get(ids_field)` outside and the rename the method `get_first_value_for_schema(list, schema)`